src/Compilers/Core/Portable/Operations/IBinaryOperatorExpression.cs

```diff
@@ -32,5 +32,13 @@ public interface IBinaryOperatorExpression : IHasOperatorMethodExpression
         /// value types.
         /// </summary>
         bool IsLifted { get; }
+        /// <summary>
+        /// <code>true</code> if this is a 'checked' binary operator.
```

Comment:
I think it would be useful to describe more about what 'checked' means semantically in the doc comment than to just repeat the API name. i.e. that it is checked for arithmetic overflow.